src/BlazorWebView/src/core/Windows/BlazorWebViewHandler.Windows.cs

```diff
@@ -45,17 +76,20 @@ private void StartWebViewCoreIfPossible()
 				throw new InvalidOperationException($"Can't start {nameof(BlazorWebView)} without native web view instance.");
 			}
 
-			var resourceAssembly = RootComponents?[0]?.ComponentType?.Assembly;
-			if (resourceAssembly == null)
-			{
-				throw new InvalidOperationException($"Can't start {nameof(BlazorWebView)} without a component type assembly.");
-			}
+			// We use the entry assembly's location to determine where the static assets got copied to.
+			// It is critical to avoid calling System.IO.Path APIs that use the "current directory"
+			// when they are passed in a relative path, because the app has no control over where
+			// the "current directory" is (it could be anywhere).
+			var entryAssemblyPath = global::System.Reflection.Assembly.GetEntryAssembly()!.Location;
```

Comment:
What's the reason for using `global::` here?

Reply:
To avoid a conflict with some sub-namespace that has `System` in it. But I think this code is gone now.